### ISSUE TITLE
Greenish colors in WSL2 Ubuntu

### ISSUE THREAD

**Issue by tangenti** [Eugeny/tabby#6184]
**Describe the problem**:
Color schemes not working correctly in WSL2 Ubuntu. Everything is kind of greenish and certainly does not match the color scheme preview in the settings.

![image](https://user-images.githubusercontent.com/48495408/163753173-42f7a9c8-45ec-4fae-b926-7f4299df86c3.png)

**To Reproduce**:
Windows Version: Windows 11 21H2 22000.61
Linux Version: Linux TANGENT 5.10.16.3-microsoft-standard-WSL2 #1 SMP Fri Apr 2 22:23:49 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
Tabby Version: Alpha 174

** Attempts **
- Change `TERM` and `COLORTERM`
- Switch Tabby terminal frontend
- Configure `PS1`, `LS_COLORS`
- Running commands related to `dircolors`

None of these worked.

**Comment by Eugeny** [comment]
No `LS_COLORS` can cause this sort of permission highlighting, is your `ls` an alias to something else?

**Comment by tangenti** [comment]
Thanks for the quick reply!
Yes, it's an alias to `exa`. The default `ls` output looks like this:
![image](https://user-images.githubusercontent.com/48495408/163778115-73e69735-bee1-4d6f-a91c-657148d7bf90.png)



As you may notice in the problem description, the output of `echo` is greenish as well :(

**Comment by Eugeny** [comment]
Is it also green with `--color=never`? Does your color theme use green for white by any chance? You can print a color table with `msgcat --color=test` (needs gettext-tools)

**Comment by tangenti** [comment]
Yes, it's still green with `--color=never`. I tried many different color schemes and got the same result.
The output that saved by the plugin shows that the unescaped output is green as well:

```
➜  ;36;96m~ ;32mecho;32;92m hello
hello
➜  ;36;96m~ ;32;92m
```
Reinstalling `zsh`, `oh-my-zsh` or Tabby itself does not solve the issue. Colors work correctly in Ubuntu and Windows Terminal.

Output of the tests:

![image](https://user-images.githubusercontent.com/48495408/163793730-3c275b3e-889b-4f5f-886b-8f053aefb10f.png)
